src/Radarr.Api.V3/swagger.json

```diff
@@ -1115,6 +1115,8 @@
                       "type": "array",
                       "items": {
                         "$ref": "#/components/schemas/History"
+                                  "infoLink": "https://wiki.servarr.com/Radarr_Settings#Custom_Formats_2",
+                                  "infoLink": "https://wiki.servarr.com/Radarr_Settings#Custom_Formats_2",
```

Comment:
Something is goofy here?

Reply:
removed...again; we'll see if it stays away